tests/_test_mapper.py

```diff
@@ -233,29 +238,29 @@ def test_coreir_ops(op):
     imap = mapper.extract_instr_map(app)
     assert len(imap) == 1
 
-@pytest.mark.parametrize("rule",Rules)
-def test_rules(rule):
-    if rule['kind'] != "1to1":
-        pytest.skip()
-    namespace,op = rule["coreir_prim"]
-    c = coreir.Context()
-    if namespace == "coreir":
-        genargs = dict(width=16)
-    elif namespace == "float":
-        genargs = dict(exp_bits=8,frac_bits=7)
-    app = make_single_op_app(c,namespace,op,**genargs)
-    app.print_()
-    mapper = LassenMapper(c)
-    for rule in Rules:
-        mapper.add_rr_from_description(rule)
-
-    try:
-        mapper.map_app(app)
-    except:
-        raise NotImplementedError("You probably need to regenerate rules by running the scripts/gen_rules.py script")
-
-    imap = mapper.extract_instr_map(app)
-    assert len(imap) == 1
+#@pytest.mark.parametrize("rule",Rules)
```

Comment:
pytest.mark.skip if you want disable the test.
Remove if its not valid

Reply:
Removed bad file and replaced it with a basic rewrite rule test for lassen